packages/devtools_app/lib/src/inspector/flutter/inspector_tree_flutter.dart

```diff
@@ -353,20 +384,31 @@ class _InspectorTreeState extends State<InspectorTree>
         child: SizedBox(
           width: controller.rowWidth + controller.maxRowIndent,
           child: Scrollbar(
-            child: ListView.custom(
-              itemExtent: rowHeight,
-              childrenDelegate: SliverChildBuilderDelegate(
-                (context, index) {
-                  final InspectorTreeRow row = controller.root?.getRow(index);
-                  return _InspectorTreeRowWidget(
-                    key: PageStorageKey(row?.node),
-                    inspectorTreeState: this,
-                    row: row,
-                  );
-                },
-                childCount: controller.numRows,
+            child: GestureDetector(
+              onTap: () {
```

Comment:
This doesn't need the closure.

```suggestion
              onTap: _focusNode.requestFocus,
```